### PR TITLE
feat: ignore originalSite if single match

### LINKED ISSUE #200: ignore "originalRepository" from discover call if only 1 result
Port from https://github.com/adobe/helix-sidekick-extension/issues/761

### PR DESCRIPTION
**Pull request by rofe**
Fix #200

### CODE REVIEW

File: src/extension/url-cache.js [hunk 1]
```diff
@@ -237,6 +237,7 @@ class UrlCache {
         );
         if (resp.ok) {
           results = await resp.json();
+          console.log('urlcache.set()', url, results);
```
Comment (dylandepass) [inline src/extension/url-cache.js:240]: Same here

**Review by dylandepass — APPROVED** [review]

File: src/extension/project.js [hunk 2]
```diff
@@ -495,11 +495,14 @@ export async function getProjectMatches(configs, tab) {
     }
   }
   if (matches.length === 0) {
-    const { owner, repo } = cachedResults.find((r) => r.originalRepository) || {};
-    if (owner && repo) {
+    console.log('---------', tab.url, ' no matches, check cache', cachedResults);
```
[the remark this answers is not in the document]
Comment (rofe) [reply src/extension/project.js:498]: LOL no

File: src/extension/url-cache.js (hunk 1 above)
Comment (rofe) [reply src/extension/url-cache.js:240]: Good catch!

**Pull request merged**